src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/tests/fields/ResourceLocator.test.js

```diff
@@ -378,6 +378,9 @@ test('Should request a new URL including the options from the ResourceFormStore
                 generationUrl: '/admin/api/resourcelocators?action=generate',
                 historyResourceKey: 'page_resourcelocators',
                 modeResolver: () => Promise.resolve('leaf'),
+                resourceStorePropertiesToRequest: {
+                    requestParamKey: 'propertyName',
```

Comment:
Shouldn't that be implemented in the other way? The property is called `resourceStorePropertiesToRequest`, so the left side here should be the name of the property from the `ResourceStore`, and right side should be the name that goes into the request.

I've done a quick check, and that's also how it works e.g. for the `routerAttributesToFormMetadata` option in the `FormViewBuilder`.

Reply:
Thats okay for me too 🙂 

Just for completion, the `selection` and `single_selection` field-type allow for the following configuration:
```
    <params>
        <param name="resource_store_properties_to_request" type="collection">
            <param name="parameterKey" value="testParameter" />
        </param>
    </params>
```

Reply:
Yeah, it's structured differently there... But when it is defined in other places the name seems mostly to read from left to right, and I think that's also what we agreed on once :slightly_smiling_face: